xcp_d/workflow/bold.py

```diff
@@ -406,6 +417,23 @@ def init_boldpostprocess_wf(lower_bpf,
         n_procs=omp_nthreads,
     )
 
+    # if presmoothing is enabled, use the presmoothed bold file as input to
+    # further preprocessing
+    # else, use the bold file from inputnode
+    bolddatanode = pe.Node(niu.IdentityInterface(
+        fields=['bold_file']),
+        name='bolddatanode')
```

Comment:
I'd recommend separating the words in the variable name with underscores (e.g., `bold_data_node`).